src/System.Collections.NonGeneric/src/System/Collections/CollectionBase.cs

```diff
@@ -33,12 +32,7 @@ protected CollectionBase(int capacity)
 
         protected ArrayList InnerList
         {
-            get
-            {
-                if (_list == null)
-                    _list = new ArrayList();
-                return _list;
-            }
+            get { return _list ?? (_list = new ArrayList()); }
```

Comment:
Please leave revert changes like this as well. I don't think they add enough value to actually warrant the change.